src/lumigo_tracer/user_utils.py

```diff
@@ -11,3 +13,38 @@ def report_error(msg: str):
     else:
         message_with_request_id = f"RequestId: {SpansContainer.get_span().function_span.get('id')} {message_with_initials}"
         print(message_with_request_id)
+
+
+def add_execution_tag(key: str, value: str, should_log_errors: bool = True) -> None:
```

Comment:
I don't think that the user knows the phrase "execution"

Reply:
The product asked to change it: we have different kinds of tags in the system so it should be `execution_id` in the platform. We want the same language in the tracer and the platform.